README.md

```diff
@@ -1,61 +1,122 @@
+We have four main expecations for community behaviours that contribute to creating a welcoming and thriving community. These are goals and aspirations to help drive us beyond just what is "ban worthy" or "legal" and towards what is actively making our community a more welcoming place. Often, it's not necessarily __what__ we're saying that is creating friction, but __how__ what we are saying is being received by the other person. 
+
+> Behave as if you were at a professional event
+
+Although we are on the internet - and perhaps even more importantly _because_ we are on the internet where tone is extremely challenging to read - choosing our words carefully and conducting yourself as a professional is important. Not sure if what you're about to say is professional? Try imagining someone in your family or friend group who's got your back, but holds you to high standards - whether that be your grandma, your cousin, our best friend or your mom - would they be cheering on what you're about to say, or would they be disappointed in your level of professionalism? 
+
+> Use kindness to build community
+
+This doesn't mean we don't hold each other to high standards for behaviour, but it does mean we call each other in (or out, if necessary) on things we are uncomfortable with in a way that is grounded in empathy and respect - even if we don't feel like we ourselves were respected in that moment. 
 
-* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
-* Exercise consideration and respect in your speech and actions. Tone on the internet is often misunderstood, and people may interpret your words differently than you might intend. So err on the side of caution.
+> Be curious to hear about the other person’s perspective
+
+Remember not to assume your experience is the default - make space for others to have different life experiences than you. One litmus test for how you are framing something is going to read as having implicit bias is to check against these three questions: 
+* *First: is this true* is what you're about to say something that is a fact, or a perspective that you hold where you can acknolwedge that others may hold different ones? Is there a way to explicitly acknolwedge that you're open to other perpsectives in how you're responding; will this read as a "yes and", or a "no because"? 
+* *Second: Is this missing something* are you encouraging all possibilities, or just the ones that come most readily to you and your life experience? What questions might you ask the other folks in the conversation to understand their experiences or motivations better? 
+* *Third: is this just* is this supporting the inclusion and equality of minority members of our community, or reinforcing barriers for those groups? This can be quite nuanced, but being explicit about who and how we want to advocate for each other, and why it matters to us, is always a great reminder. 
```

Comment:
```suggestion
* *Third: is this just?*  Is this supporting the inclusion and equality of minority members of our community, or reinforcing barriers for those groups? This can be quite nuanced, but being explicit about who and how we want to advocate for each other, and why it matters to us, is always a great reminder. 
```

TBH I'm struggling with the word `just` in that, but I can't think of a better word. 🤷  Maybe `fair`, but that doesn't have as much weight.

Reply:
Yeah, I had to read it a few times, because I was reading just as **only**, as opposed to **equitable**. 

I wonder if this could be improved by replacing `this` with `your framing`. 

It does make each of the questions in the litmus more wordy, but I feel like it may add clarity.

Reply:
This part was a combination of [this article](https://dailystoic.com/dont-make-assumptions/) and one of the suggested responses to the prompts during the interviews that was similar in nature (the is it fair/is it kind/is it true). 

I agree the word "just" is weird, but I also appreciate the nod to justice/equality. I like your suggestion jamie, and I'm pushing up an overall modification to this paragraph. Please let me know if it reads better!